### ISSUE TITLE
Text Contrast in Minify Warning Not WCAG-Compliant 

### ISSUE THREAD

**Issue by jessecowens** [BoldGrid/w3-total-cache#62]
**Is your feature request related to a problem? Please describe.**
The important warning text that is presented when a user clicks "Enable" in the Minify settings has insufficient text size/color contrast to be adequately readable. The background is off-white #FDFDFD and the text color is #88898A
https://webaim.org/resources/contrastchecker/?fcolor=88898A&bcolor=FDFDFD

This leads to frustrating user experience and possible accessibility issues attempting to read the warning text, possibly causing the user to ignore the text altogether.

**Describe the solution you'd like**
In order to meet WCAG guidelines, the text should have a color contrast of at least 4.5:1 at its current size of 13px. A text-color of #757575 would be sufficient to meet guidelines.

**Describe alternatives you've considered**
In order to comply with WCAG guidelines without changing the color, the text would need to be increased to 14pt bold.

**Additional context**
![Selection_183](https://user-images.githubusercontent.com/8019842/66417491-54a98f80-e9bd-11e9-84c3-82c8f99e6b5d.png)

**Comment by jacobd91** [comment]
Thanks for reporting this and for including the specific contrast measurements. The Minify warning shown in the original report is no longer part of the current W3 Total Cache interface; Minify is now enabled through the standard WordPress admin controls, and the cited color combination is no longer used. I’m closing this as no longer applicable. If you notice a contrast problem in the latest version, please open a new issue with a screenshot and the affected screen so we can take another look.
